### PR TITLE
added author info

### PR DESCRIPTION
**Pull request by bruhad-dave**

### CODE REVIEW

**Review by bruhad-dave — COMMENTED** [review]
fixing formatting on the authors section

**Review by bruhad-dave — COMMENTED** [review]
formatting fix - second attempt, adding non-breaking space character

**Review by bruhad-dave — COMMENTED** [review]
format fixing attempt 3, removed spacing before hyperlink end tag

File: src/components/header.js [hunk 1]
```diff
@@ -297,6 +297,15 @@ const AboutContent = ({ visible }) => {
               </span>
             </li>
           </ul>
+          <strong className='d-block mb-3'>Authors and Contributors</strong>
+              <span className='d-block'>
+                <p>
+                  <a href='https://github.com/johnlees' >Dr John Lees</a>, 
+                  <a href='https://github.com/Danderson123' >&nbsp;Daniel Anderson</a>, and 
+                  <a href='https://github.com/bruhad-dave' >&nbsp;Bruhad Dave 
+                  </a>
```
Comment (johnlees) [inline src/components/header.js:306]: I think you want to exclude the spaces from the hyperlinks:
```suggestion
                  <a href='https://github.com/johnlees' >Dr John Lees</a>, 
                  &nbsp;<a href='https://github.com/Danderson123' >Daniel Anderson</a>, and 
                  &nbsp;<a href='https://github.com/bruhad-dave' >Bruhad Dave</a>
```

**Review by johnlees — CHANGES_REQUESTED** [review]
A very minor change, more to give you an example of code review. If you make this change, check it looks ok, then request me again as reviewer, I will merge

**Comment by bruhad-dave** [comment]
made minor changes to whitespaces around hyperlinks

**Review by bruhad-dave — COMMENTED** [review]
moved nbsc char to after the hyperlink end tag rather than before the start tag

**Review by johnlees — APPROVED** [review]

**Pull request merged**